src/getPublicKeys.h

```diff
@@ -0,0 +1,33 @@
+#ifndef H_CARDANO_APP_GET_PUB_KEYS
+#define H_CARDANO_APP_GET_PUB_KEYS
+
+#include "common.h"
+#include "handlers.h"
+#include "bip44.h"
+#include "keyDerivation.h"
+
+#define MAX_PUBLIC_KEYS 1000
+
+typedef enum {
+	GET_KEYS_STAGE_NONE = 0,
+	GET_KEYS_STAGE_INIT = 20,
+	GET_KEYS_STAGE_GET_KEYS = 40,
+	GET_KEYS_STAGE_SINGLE_KEY = 60 // terminal state for the case where a single key is asked
```

Comment:
I don't see this stage being referenced anywhere else, is it just a leftover from the previous version?